100.md

```diff
@@ -0,0 +1,70 @@
+
+NIP-100
+======
+
+Lock user
+-------------
+
+`draft` `optional`
+
+### Abstract
+
+In Nostr, users are defined by a pair of public and private keys. These keys are what allow the user to be identified (through the known public key) and ensure that the content is published by the owner of the key pair (through signing with the private key).
+
+This cryptography-based mechanism is very robust and secure, but it is not immune to being attacked. One of the main attack vectors is the theft of the private key itself.
+
+Unlike other protocols like Bitcoin, the theft of a private key does not imply a direct theft of the user's assets, but it can have other implications such as identity theft and consequent fraud attempts. This has a special impact when the stolen key belongs to a highly relevant user.
+
+Although there are currently mechanisms to protect and safeguard our keys, this does not provide complete reliability, and they can always be stolen. To try to mitigate the impact of this problem, this NIP provides a mechanism by which once the key owner suspects or has certainty that the keys have been stolen, they can lock their user and thus prevent (or at least partially prevent) events from being emitted in their name.
+
+### User Lock Event
+
+#### Field `kind`
+
+The `kind:1000` event is defined, which will be sent to the Relays and will indicate that this user should be locked.
+
+#### Field `pubkey`
+
+This is the public key of the event signer and the corresponding public key associated with the stolen private key that wants to be locked.
+
+#### Field `content`
+
+The `content` field may include a description of the reason why the user wants to be locked, although it can be left empty.
```

Comment:
I think `content` should be blank. Clients and relays can't distinguish between your `kind 1000` event and one that has been created by an attacker. Better not give the attacker a chance to publish a malicious message.

Reply:
Yes, I defined it as optional in case the user wanted to indicate a message to their followers, but you are right. If the attacker, for example, locks the key before the real user, and specifies in the `.content` a message such as a new fake public key, it can be a new attack vector.

I change it